candies/cli/parsers/standard.py

```diff
@@ -91,9 +115,9 @@ def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
         subparsers = parser.add_subparsers(dest=prefix + 'command')
 
         for name, subcommand in command.subcommands.items():
-            subparser = subparsers.add_parser(name)
+            _ = configured(new_subparser(subparsers), subcommand, prefix + '.')
```

Comment:
Is `_` just to show that the `configured` returns something not needed?

Reply:
Yes.